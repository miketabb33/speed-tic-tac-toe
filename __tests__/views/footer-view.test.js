import React from 'react'
import renderer from 'react-test-renderer'
import FooterView from '../../views/footer-view'

test('render', () => {
	const subject = renderer.create(
		<FooterView />
	)

	const result = subject.toJSON()
	expect(result).toMatchSnapshot()
})