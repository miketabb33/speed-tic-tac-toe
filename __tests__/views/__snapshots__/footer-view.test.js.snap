// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render 1`] = `
<div
  className="footerBar"
>
  <div
    style={
      Object {
        "boxSizing": "border-box",
        "display": "inline-block",
        "margin": 0,
        "maxWidth": "100%",
        "overflow": "hidden",
        "position": "relative",
      }
    }
  >
    <div
      style={
        Object {
          "boxSizing": "border-box",
          "display": "block",
          "maxWidth": "100%",
        }
      }
    >
      <img
        alt=""
        aria-hidden={true}
        src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iNjAiIGhlaWdodD0iNjAiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgdmVyc2lvbj0iMS4xIi8+"
        style={
          Object {
            "border": "none",
            "display": "block",
            "margin": 0,
            "maxWidth": "100%",
            "padding": 0,
          }
        }
      />
    </div>
    <img
      data-nimg="intrinsic"
      decoding="async"
      draggable="false"
      src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
      style={
        Object {
          "border": "none",
          "bottom": 0,
          "boxSizing": "border-box",
          "display": "block",
          "height": 0,
          "left": 0,
          "margin": "auto",
          "maxHeight": "100%",
          "maxWidth": "100%",
          "minHeight": "100%",
          "minWidth": "100%",
          "objectFit": undefined,
          "objectPosition": undefined,
          "padding": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
          "width": 0,
        }
      }
    />
    <noscript>
      <img
        data-nimg="intrinsic"
        decoding="async"
        draggable="false"
        loading="lazy"
        src="/_next/image?url=%2Flogo.jpg&w=128&q=75"
        srcSet="/_next/image?url=%2Flogo.jpg&w=64&q=75 1x, /_next/image?url=%2Flogo.jpg&w=128&q=75 2x"
        style={
          Object {
            "border": "none",
            "bottom": 0,
            "boxSizing": "border-box",
            "display": "block",
            "height": 0,
            "left": 0,
            "margin": "auto",
            "maxHeight": "100%",
            "maxWidth": "100%",
            "minHeight": "100%",
            "minWidth": "100%",
            "objectFit": undefined,
            "objectPosition": undefined,
            "padding": 0,
            "position": "absolute",
            "right": 0,
            "top": 0,
            "width": 0,
          }
        }
      />
    </noscript>
  </div>
  <div
    className="footerText"
  >
    Speed Tic Tac Toe
  </div>
</div>
`;
